python/experimental/tests/ast/test_library_kernels.py

```diff
@@ -12,10 +12,15 @@
 
 import cudaq
 
-cudaq.enable_jit()
-from cudaq.lib import fermionic_swap, givens 
+# [RFC]: 
+# FIXME: Why does having this here instead of inside the respective test cause issues?
```

Comment:
I believe it was because of the multi-threaded nature of Pytest.

Reply:
Well, yes, but I think we need to make the multithreaded nature work.